### ISSUE TITLE
Ionic v0.1.0

### ISSUE THREAD

**Issue by 0fatihyildiz** [oku-ui/pergel#202]
### MVP Roadmap for Ionic Pergel Templates and Pergel Config 🔥

- [ ] Capacitor Config Setup
    - [ ] Pergel Connect Capacitor Plugins
- [ ] Splash Screen 
- [ ] Onboarding
    - [ ] Permission Access 
    - [ ] Presentation (Slider)
- [ ] Auth
    - [ ] Sign in
    - [ ] Sign Up
    - [ ] Forgot Password
    - [ ] OAuth (Google, Github)
- [ ] Home
- [ ] Settings
- [ ] Local Notification
- [ ] Cloud Notification
- [ ] RevenueCat
- [ ] Privacy Policy
- [ ] Terms of Usage
- [ ] Automatically Deploy CI CD
    - [ ] Appstore
    - [ ] Playstore
- [ ] Multi Themes
- [ ] Nuxt Ionic Pages
- [ ] Nuxt Ionic Helper Composables
- [ ] TailwindCSS Support
- [ ] Automatically Version Upgrade

**Comment by Aksoom-Hussain** [comment]
instead of going with ionic UI you can directly use the capacitor.js to scaffold the mobile responsive component and manage  dynamic theme

**Comment by productdevbook** [comment]
> instead of going with ionic UI you can directly use the capacitor.js to scaffold the mobile responsive component and manage dynamic theme

Yes We are also building a capacitor module. This way you will only activate the capacitor and use the module you want.

`capacitor: true`

**Comment by productdevbook** [comment]
mobile app dont used ssr. `ssr:false` default. How do you plan to use SSR in the mobile app ?

**Comment by Aksoom-Hussain** [comment]
> mobile app dont used ssr. `ssr:false` default. How do you plan to use SSR in the mobile app ?

ssr is not possible in capacitor local build (sending code in build) but using directly with hosted domain but got issues around the native components as if call pwa components from plugin
